orchagent/orchdaemon.cpp

```diff
@@ -304,7 +324,17 @@ void OrchDaemon::start()
         }
 
         auto *c = (Executor *)s;
-        c->execute();
+
+        if (restored)
```

Comment:
>if [](start = 8, length = 2)

select() is used for normal processing, and it is not for warm start, because all the data is already added during bake() -> addExistingData().

You can extract the warm start logic and move before the normal processing while-loop.

Reply:
Right.  Previously configDb data was retrieved here,  it is not needed any more.  pre-shutdown  warm restart check may need to be processed in select(), and possibly arp sync up event too,  but that could be done in separate PRs.